program/rust/src/processor/upd_price.rs

```diff
@@ -126,14 +163,79 @@ pub fn upd_price(
         }
     }
 
+    let mut price_data = load_checked::<PriceAccount>(price_account, cmd_args.header.version)?;
+    // We want to send a message every time the aggregate price updates. However, during the migration,
+    // not every publisher will necessarily provide the accumulator accounts. The message_sent_ flag
+    // ensures that after every aggregate update, the next publisher who provides the accumulator accounts
+    // will send the message.
```

Comment:
How are you planning to send prev_timestamp_? Since it is potentially not available anymore when the next publisher comes.